source/atlas-cli-deploy-local.txt

```diff
@@ -86,6 +86,105 @@ Before you begin, complete the following prerequisites:
 
 .. include:: /includes/steps-atlas-cli-deploy-local-prereqs.rst
 
+.. _atlas-cli-pvt-registry-setup:
+
+(Optional) Set up a Private Image Registry 
+------------------------------------------
+
+This section demonstrates how to pull and tag images, load images to a
+container registry, and configure the {+atlas-cli+} to use private
+images for building local clusters. 
+
+.. procedure:: 
+   :style: normal 
+
+   .. step:: Install the Docker command line tools.
+   
+   .. step:: Create your container registry with the namespace and name of repository. 
+
+   .. step:: Fetch the MongoDB image for local deployment. 
+
+      To fetch, run the following command after replacing the following
```

Comment:
~fetch~ pull and re-tag

It might also be helpful here to link to https://hub.docker.com/r/mongodb/mongodb-atlas-local/tags